scripts/importing-Exporting-Config.js

```diff
@@ -0,0 +1,480 @@
+            var team_Specific_Data_Entry = curr_Event_Entries[curr_Team_Key];
+            var team_Specific_Data_Keys = Object.keys(team_Specific_Data_Entry);
+            //console.log(team_Specific_Data_Keys);
+
+            for (var k = 0; k < team_Specific_Data_Keys.length; k++)
+            {
+                var match_Or_Pit_Key = team_Specific_Data_Keys[k];
+                var match_Or_Pit_Entry = team_Specific_Data_Entry[match_Or_Pit_Key];
+                //console.log(match_Or_Pit_Key);
+
+                if (match_Or_Pit_Key.includes('match data'))
+                {
+                    //has an array full of elment/frequency pair objects in it for every match
+                    var num_Matches = match_Or_Pit_Entry.length;
+
+                    for (var m = 0; m < num_Matches; m++)
+                    {
+                        var curr_Match_Arr = match_Or_Pit_Entry[m];
+
+                        //need to find match num to set a local storage object, {'Match #' : number} is the first object in the array
+                        var match_Num_Obj = curr_Match_Arr[0];
+                        var match_Num_Key = Object.keys(match_Num_Obj)[0]; //only of length 1
+                        var match_Num = match_Num_Obj[match_Num_Key];
+
+                        var local_Storage_Obj_Name = matchAnswerObjName(season,curr_Event,curr_Team_Num,match_Num);
+                        var local_Storage_Obj_Value = JSON.stringify(curr_Match_Arr);
+                        localStorage.setItem(local_Storage_Obj_Name,local_Storage_Obj_Value);
+                    }
+                }
+
+                if (match_Or_Pit_Key.includes('pit data'))
+                {
+                    var local_Storage_Obj_Value = JSON.stringify(match_Or_Pit_Entry); //is just an array with Q/A pair objects in it
+                    var local_Storage_Obj_Name = pitAnswerObjName(season,curr_Event,curr_Team_Num);
+                    localStorage.setItem(local_Storage_Obj_Name,local_Storage_Obj_Value);
+                }
+            }
+            cookie_Team_Obj[curr_Team_Num] = curr_Team_Name;
+        }
+
+        //set cookie for teams in every event
+        var cookie_Name = event_data_cookie_name(season,curr_Event);
+        var cookie_Value = JSON.stringify(cookie_Team_Obj);
+        setCookie(cookie_Name,cookie_Value,750);
+    }
+}
+
+function showImportModal(object)
+{
+    var keys = Object.keys(object);
+    var seasonList = object[EXPORT_CONFIG_SEASON_LIST_KEY]; // season config list is the entry for the first key
+    
+    //add seasons to modal text
+    var modalText = 'This file contains data that will overwrite the following season(s): <br/><br/>';
+    for (var i = 0; i < seasonList.length; i++)
+    {
+        if (i == seasonList.length-1)
+        {
+            modalText += `${seasonList[i]}.<br/> `
+        } else
+        {
+            modalText += `${seasonList[i]},<br/> `;
+        }
```

Comment:
90% of the code is the same between the if and the else. Make the part that's different obvious (ideally by having it be the only part of the if statement). Might look like:

```js
modalText += seasonList[i];
if(i == seasonList.lenth - 1) {
  modelText += '.';
} else {
 modalText += ',';
}

modalText += '<br/>';
```

When you break it down like that, it looks like you could even use a ternary operator (which is similar to the LabVIEW select).

```js
modalText += (i == seasonList.length - 1)?'.':',';
```